spec/delayed_job_worker_pool/registry_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+
+describe DelayedJobWorkerPool::Registry do
```

Comment:
Can we add test coverage for edge cases in this class e.g. duplicate group names, looking up workers that don't exists, etc.?